Rebus.RabbitMq.Tests/RabbitMqHeaderTest.cs

```diff
@@ -60,7 +60,7 @@ void Callback(object sender, BasicReturnEventArgs eventArgs)
             var bus = StartOneWayClient(Callback);
             await bus.Advanced.Routing.Send(_noneExistingQueueName, "I have headers", headers);
 
-            gotCallback.WaitOrDie(TimeSpan.FromSeconds(2));
+            gotCallback.WaitOrDie(TimeSpan.FromSeconds(5000));
```

Comment:
Do you really mean to wait for up to ~1.5 hours for this? 😀

Reply:
No, i was debugging something :P